src/syntax-highlight/syntax-highlight.js

```diff
@@ -39,10 +39,10 @@ export default function syntaxHighlight(diff, afterWordDiff) {
     }
 
     const $diff = $(diff)
-    syntaxHighlightSourceCodeLines($diff)
+    syntaxHighlightSourceCodeLines($diff, 'pre.source:not([class*=language])')
 
     afterWordDiff(() => {
-        syntaxHighlightSourceCodeLines($diff)
+        syntaxHighlightSourceCodeLines($diff, '.addition pre, .deletion pre')
```

Comment:
second time, we will want to look for tags `pre:has(ins)` `pre:has(del)` but if word diff are only in `addition` block, the `deletion` block will be rendered as well, so we have to re-highlight there too. maybe you know how to combine, this one above with and a condition at least `has(ins) or has(del)`

Reply:
![image](https://user-images.githubusercontent.com/23088305/81894154-1d1b4b00-957d-11ea-84dd-9e36fd7b85a8.png)